libkbfs/keybase_daemon_rpc.go

```diff
@@ -308,9 +315,35 @@ func (k *KeybaseDaemonRPC) ShouldRetryOnConnect(err error) bool {
 	return !inputCanceled
 }
 
+func (k *KeybaseDaemonRPC) keepAliveLoop(ctx context.Context) {
+	// We need to send another HelloIAm message as soon as the
```

Comment:
This comment is a bit confusing. The main problem is that we can't detect if a connection is dropped until the next RPC, right? Maybe word it something like:

```
If the connection is dropped, we need to re-connect and send another HelloIAm
 message. However, we can't actually detect when the connection is closed until
 KBFS makes another outgoing RPC, ...
```